zulipterminal/config/keys.py

```diff
@@ -317,8 +317,13 @@ class KeyBinding(TypedDict):
     },
     'USER_INFO': {
         'keys': ['i'],
-        'help_text': 'Show/hide user information (from users list)',
-        'key_category': 'general',
+        'help_text': 'Show/hide user information',
+        'key_category': 'user_list',
+    },
+    'NARROW_TO_USER_PM': {
```

Comment:
This one is a bit tricky because we don't actually use this specific key, but if I understand it correctly the functionality to narrow to a DM happens through the fact that the user list is composed of `UserButton`s, which is a `TopButton`, which maps `ENTER` to `activate` [here](https://github.com/zormit/zulip-terminal/blob/e5e9010850ab4c678890686e3707d7e3e842385e/zulipterminal/ui_tools/buttons.py#L123).

I would suggest to leave a comment here to explain this in fewer words than I do, in order to prevent confusing future readers of this code. Maybe something like "this key is only for documenting the functionality, but the functionality happens through a generic ENTER"?

Reply:
@Niloth-p it seems you have now added a comment for `USER_INFO`, which might make sense as well to some extent, but that one is actually used to retrieve its key, e.g. in `zulipterminal/ui_tools/buttons.py`.

Whereas `NARROW_TO_USER_PM` is not used anywhere and just lives here for the documentation/help-text.

Reply:
Haha, thanks for the catch. I can't believe I accidentally wrote the comment for the wrong command.

> which might make sense as well to some extent

No, the comment was intended for this command, I wrote it in another text editor and I pasted it into the wrong command. The help text mentions that it clarifies that it does not open the user profile info, and I pasted it for the user info command lol.

Thank you for the save!